shared/templates/rsyslog_logfiles_attributes_modify/ansible.template

```diff
@@ -51,7 +51,7 @@
 
 - name: '{{{ rule_title }}} - Extract log files new format'
   ansible.builtin.shell: |
-    set -o pipefail
+    {{% if not 'debian' in product %}}set -o pipefail{{% endif -%}}
```

Comment:
The default shell in Debian for scripts is dash. dash doesn't have the pipefail option.

Reply:
Thanks for a great explanation.

I have found that at this moment we have in our CI relaxed this Ansible lint check to a warning, due to some other existing ules that don't have the pipefail set in their Ansible code. https://github.com/ComplianceAsCode/content/blob/ae3019d770b55576459197a5944aa3532879ed76/.config/ansible-lint.yml#L15-L22

Also, there is a complex discussion about it on the Ansible lint GitHub https://github.com/ansible/ansible-lint/issues/497 and there is no specific outcome.

So after finding these information I would prefer to keep this `if` condition here as you proposed. It would be nice to add an explanatory comment to the code there.

Reply:
I will add an explanation ASAP.